doc/nrf/external_comp/bt_fast_pair/index.rst

```diff
@@ -0,0 +1,20 @@
+.. _ug_bt_fast_pair:
+
+Google Fast Pair integration
+############################
+
+Google Fast Pair is a standard for pairing *Bluetooth®* and Bluetooth Low Energy (LE) devices with as little user interaction required as possible.
+Google also provides additional features built upon the Fast Pair standard.
+For detailed information about supported functionalities, see the official `Fast Pair`_ documentation.
+
+The subpages include additional pages that discuss the Fast Pair helper modules.
+While these helper modules can provide additional functionality in your application, they are not strictly required for the Fast Pair integration.
+
+.. note::
+   The software maturity level for the Fast Pair support in the |NCS| is indicated at the use case level in the :ref:`Google Fast Pair use case support <software_maturity_fast_pair_use_case>` table and at the feature level in the :ref:`Google Fast Pair feature support <software_maturity_fast_pair_feature>` table.
+
+.. toctree::
+   :maxdepth: 1
+   :glob:
+
+   *
```

Comment:
Can we change the order so that the Integration overview comes first and then the Advertising Manager integration.

Reply:
Good point, Pekka. I have made the changes.